MCPTT/UI/CoversationView/View/ChatMessageCell.swift

```diff
@@ -13,21 +13,24 @@ class ChatMessageCell: UICollectionViewCell {
     @IBOutlet weak var messageSender: UILabel?
     @IBOutlet weak var messageDate: UILabel?
     @IBOutlet weak var messageTime: UILabel?
-    @IBOutlet weak var commentImageView: UIImageView!
-    @IBOutlet weak var messageDateHeightConst: NSLayoutConstraint!
-  //  @IBOutlet weak var rightSideTimeWidthConst: NSLayoutConstraint!
-    @IBOutlet weak var commentImageHeightConst: NSLayoutConstraint!
     
+    @IBOutlet weak var commentImageView: UIImageView!
     @IBOutlet weak var profileView: UIImageView!
-  //  @IBOutlet weak var LeftSideTimeWidthConst: NSLayoutConstraint!
     @IBOutlet weak var textView: UITextView!
     
+    @IBOutlet weak var textViewWidthConst: NSLayoutConstraint!
+    @IBOutlet weak var imageViewLeadingConst: NSLayoutConstraint!
+    @IBOutlet weak var imageViewTrailingCont: NSLayoutConstraint!
+    @IBOutlet weak var commentViewWidthConst: NSLayoutConstraint!
+    
     override func awakeFromNib() {
         super.awakeFromNib()
     }
     
     public func configure(senderName: String, senderDate: Date, messageText: String) {
         
+          textView?.text      = messageText
```

Comment:
even if swiftlint doesn't give error here, keep only one space before and after '=' sign. And apply it to other places too.